src/lesson/lesson03.js

```diff
@@ -884,6 +900,9 @@ function Lesson03() {
     )
   );
 
+  commandsReference = [commandsReference,
```

Comment:
I think you meant `commandsReference.push(Constants.Reference.COMPONENT_SENSOR)`. The current code creates a list with two elements: a list and the constant. It may work because of the way rendering is implemented, however.